src/main/java/taub/gameoflife/GameFrame.java

```diff
@@ -35,35 +39,40 @@ public GameFrame()
 
         setLayout(new BorderLayout());
 
-        add(grid);
         add(buttonPanel, BorderLayout.SOUTH);
 
-        grid.changeField(18, 4);
-        grid.changeField(18, 5);
-        grid.changeField(18, 6);
-
-
-        // setup actions
-        playButton.addActionListener(new ActionListener()
+        try
         {
-            @Override
-            public void actionPerformed(ActionEvent e)
+            Path p = Paths.get(ClassLoader.getSystemResource("glider.rle").toURI());
+            File file = new File(String.valueOf(p));
+            RleReader reader = new RleReader(file, grid);
+            grid = reader.getFileArr();
```

Comment:
Why are you passing then returning grid?

Reply:
I got rid of the returning statement, but I need it because I use the changeField method from my grid class to make cells alive as I read them from the rle file.